### ISSUE TITLE
BUG: Resizing panes is slow and doesn't really follow the mouse

### ISSUE THREAD

**Issue by seangates** [storybooks/storybook#3887]
If you are reporting a bug or requesting support, start here:
### Bug or support request summary

This is my first time using Storybook with a project that I've been working on, but I wanted to resize the "knobs" pane and when I clicked then dragged the pane the divider did not move with my mouse. Instead, it just moved a couple of pixels. So I tried again, and again it only moved a couple of pixels at a time.

### Steps to reproduce

1. visit https://nm123github.github.io/ebayui-core/
2. try to resize the "Knobs" pane to make it larger

https://www.useloom.com/share/e92710e1ae8b47d284b854ee88b3f747 (Sorta hard to follow when Loom doesn't follow my mouse after I click. But watch where my mouse ends up after I click and drag, way up the window)

### Please specify which version of Storybook and optionally any affected addons that you're running

- @storybook/react 4.0.0-alpha.11
- @storybook/addon-something x.x.x

### Affected platforms 

- Chrome for Mac

### Screenshots / Screencast / Code Snippets (Optional)

https://www.useloom.com/share/e92710e1ae8b47d284b854ee88b3f747

**Comment by igor-dv** [comment]
Looks strange 🤔. In the latest alpha it does work very smooth:

https://storybooks-official.netlify.com/?selectedKind=Addons%7CKnobs.withKnobs&selectedStory=tweaks%20static%20values&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybooks%2Fstorybook-addon-knobs

**Comment by igor-dv** [comment]
BTW, Isn't it a `marko`?

**Comment by seangates** [comment]
Yup.

**Comment by igor-dv** [comment]
@seangates, I've checked again your example (wanted to run a profiler on it), and it looks now fast. Did you find a fix in your code? Or did the upgrade help?

**Comment by seangates** [comment]
@igor-dv Thanks for looking into it! Yes, it works _much_ better now.

BTW, that's not my project, but one that I'm affiliated with. I believe the developer did the update. Let me know if you want to do any further investigation. Otherwise, I'm closing this.
